utils/exceptions.cc

```diff
@@ -78,6 +78,8 @@ void set_abort_on_internal_error(bool do_abort) {
 void on_internal_error(seastar::logger& logger, const seastar::sstring& msg) {
     if (abort_on_internal_error.load()) {
         logger.error("{}, at: {}", msg.c_str(), seastar::current_backtrace());
+        std::cout.flush();
+        std::cerr.flush();
```

Comment:
How about when we use the syslog backend? Or does that also use `std::cout/std::cerr`? Maybe we should have a `logger.flush()` or something similar.

Reply:
Sent scylladb/seastar#675 to define logger::flush and call it in the seastar abort signal handler path so this won't be needed.